src/utils/tsdHelper.ts

```diff
@@ -49,31 +48,31 @@ export class TsdHelper {
                     .then(() => installedTypeDefs.push(dest));
             }));
         })
-            .finally(() => {
-                if (installedTypeDefs.length === 0) return;
-
-                let typingsFolder = path.resolve(projectRoot, TsdHelper.USER_TYPINGS_FOLDERNAME);
-                let indexFile = path.resolve(typingsFolder, "cordova-typings.d.ts");
-
-                // Ensure that the 'typings' folder exits; if not, create it
-                if (!CordovaProjectHelper.existsSync(typingsFolder)) {
-                    CordovaProjectHelper.makeDirectoryRecursive(typingsFolder);
-                }
-
-                let references = CordovaProjectHelper.existsSync(indexFile) ? fs.readFileSync(indexFile, "utf8") : "";
-                let referencesToAdd = installedTypeDefs
-                    // Do not add references to typedefs that are not exist,
-                    // this rarely happens if typedef file fails to copy
-                    .filter(typeDef => CordovaProjectHelper.existsSync(typeDef))
-                    .map(typeDef => path.relative(typingsFolder, typeDef))
-                    // Avoid adding duplicates if reference already exist in index
-                    .filter(typeDef => references.indexOf(typeDef) < 0)
-                    .map(typeDef => `/// <reference path="${typeDef}"/>`);
-
-                if (referencesToAdd.length === 0) return;
-
-                fs.writeFileSync(indexFile, [references].concat(referencesToAdd).join("\n"), "utf8");
-            });
+        .finally(() => {
+            if (installedTypeDefs.length === 0) return;
+
+            let typingsFolder = path.resolve(projectRoot, TsdHelper.USER_TYPINGS_FOLDERNAME);
+            let indexFile = path.resolve(typingsFolder, "cordova-typings.d.ts");
+
+            // Ensure that the 'typings' folder exits; if not, create it
+            if (!CordovaProjectHelper.existsSync(typingsFolder)) {
+                CordovaProjectHelper.makeDirectoryRecursive(typingsFolder);
+            }
+
+            let references = CordovaProjectHelper.existsSync(indexFile) ? fs.readFileSync(indexFile, "utf8") : "";
+            let referencesToAdd = installedTypeDefs
+                // Do not add references to typedefs that are not exist,
+                // this rarely happens if typedef file fails to copy
+                .filter(typeDef => CordovaProjectHelper.existsSync(typeDef))
+                .map(typeDef => path.relative(typingsFolder, typeDef))
+                // Avoid adding duplicates if reference already exist in index
+                .filter(typeDef => references.indexOf(typeDef) < 0)
+                .map(typeDef => `/// <reference path="${typeDef}"/>`);
+
+            if (referencesToAdd.length === 0) return;
+
+            fs.writeFileSync(indexFile, [references].concat(referencesToAdd).join("\n"), "utf8");
```

Comment:
`"utf8"` is a [default](https://nodejs.org/api/fs.html#fs_fs_writefilesync_file_data_options) value. Maybe we may skip it here?

```suggestion
            fs.writeFileSync(indexFile, [references].concat(referencesToAdd).join("\n"));
```